frontend/src/components/ProductDetail.jsx

```diff
@@ -1,9 +1,14 @@
+// Import React tools we need
 import { useState } from 'react'
+import './ProductDetail.css'
 
+// This component shows detailed information about a single product
 function ProductDetail({ product, onBack }) {
-  const [quantity, setQuantity] = useState(1)
-  const [addedToCart, setAddedToCart] = useState(false)
+  // State variables to track what user is doing
+  const [showAllIngredients, setShowAllIngredients] = useState(false) // Should we show all ingredients?
```

Comment:
please make the comments descriptive, and you don't need question mark in your comment